tiledb/sm/config/config.cc

```diff
@@ -59,8 +59,9 @@ const std::string Config::SM_CHECK_COORD_DUPS = "true";
 const std::string Config::SM_CHECK_COORD_OOB = "true";
 const std::string Config::SM_CHECK_GLOBAL_ORDER = "true";
 const std::string Config::SM_TILE_CACHE_SIZE = "10000000";
-const std::string Config::SM_MEMORY_BUDGET = "5368709120";       // 5GB
-const std::string Config::SM_MEMORY_BUDGET_VAR = "10737418240";  // 10GB;
+const std::string Config::SM_MEMORY_BUDGET = "5368709120";           // 5GB
+const std::string Config::SM_MEMORY_BUDGET_VAR = "10737418240";      // 10GB;
+const std::string Config::SM_MEMORY_BUDGET_VALIDITY = "5368709120";  // 5GB;
```

Comment:
Do we really need this new config parameter? Similar to offsets, the validity vector is always fixed-sized. In that sense, we could consider its budget being `Config::SM_MEMORY_BUDGET`, similar to the offsets.

Reply:
That's reasonable, I'll remove this parameter.